README.md

```diff
@@ -1,4 +1,6 @@
-# Gnosis Safe Android App [![Build Status](https://travis-ci.org/gnosis/safe-android.svg?branch=master)](https://travis-ci.org/gnosis/safe-android)
+# Gnosis Safe Android App 
+[![Build Status](https://travis-ci.org/gnosis/safe-android.svg?branch=master)](https://travis-ci.org/gnosis/safe-android)
```

Comment:
we don't need travis

Reply:
What do we need this line for?

Reply:
Removed in e3d6fa7ee4d901fb59cc67e34249d9bd4300ab8d